modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/stat/StatsKey.java

```diff
@@ -15,12 +15,13 @@
  */
 package org.apache.ignite.internal.processors.query.stat;
 
+import java.io.Serializable;
 import java.util.Objects;
 
 /**
  * Statistics key.
  */
-public class StatsKey {
+public class StatsKey implements Serializable {
     /** Object schema. */
```

Comment:
static long serialVersionUid  is mandatory.

Reply:
Done